__test__/index.test.js

```diff
@@ -1,5 +1,4 @@
-const csc = require('../index.js');
-
+const csc = require('../index.ts');
```

Comment:
could be replaced by the import syntax instead `import csc from '../index.ts;'

Reply:
ok, for that I had to convert my .js test files to .ts and run build command again.
Is it what you expected?

Reply:
yes that and the import syntax you adopted in your last commit, thanks!